lib/topological_inventory/orchestrator/config_map.rb

```diff
@@ -272,6 +272,7 @@ def source_to_yaml(source)
           :path           => source.endpoint['path'],
           :receptor_node  => source.endpoint['receptor_node'],
           :account_number => source.tenant,
+          :image          => source.source_type['collector_image'],
```

Comment:
Should this naming designate the `sha` - I believe 'collector_image' is detailed, but to help understand that key expects a sha tag vs just 'any' tag could potentially make the code more readable.

Reply:
I'm down for that change - but can I do it in a followup? I would have to regenerate the test-data digests which is pretty time consuming.

Reply:
Updated it to `:image_tag_sha`